pkg/monitor/cluster/healthz.go

```diff
@@ -21,5 +21,21 @@ func (mon *Monitor) emitAPIServerHealthzCode(ctx context.Context) (int, error) {
 		"code": strconv.FormatInt(int64(statusCode), 10),
 	})
 
+	return err
+}
+
+func (mon *Monitor) getAPIServerPingCode(ctx context.Context) (int, error) {
+	var statusCode int
+	err := mon.cli.Discovery().RESTClient().
+		Get().
+		AbsPath("/healthz/ping").
+		Do(ctx).
+		StatusCode(&statusCode).
```

Comment:
is there any reference to these status codes?

Reply:
We emit a metric so we can track what the status code is.